.github/workflows/run-style-checks.yml

```
@@ -0,0 +1,28 @@
+name: Style check
+
+on:
+  pull_request
+
+jobs:
+  style:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.8']
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies and local packages
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8
+        python -m pip install flake8-ets
+    - name: Run style checks
+      run: |
+        python -m flake8
```

nitpicky suggestion : empty line at end of file.

```suggestion
        python -m flake8

```